docs/index.md

```diff
@@ -2,7 +2,23 @@
 
 _Take control of your internal daemons!_
 
-**Pebble** helps you to orchestrate a set of local service processes as an organised set. It resembles well known tools such as _supervisord_, _runit_, or _s6_, in that it can easily manage non-system processes independently from the system services, but it was designed with unique features that help with more specific use cases.
+**Pebble** is an open-source, declarative, client/server service manager that helps you orchestrate a set of local processes as an organised set on UNIX-like operating systems. It resembles well-known tools such as _supervisord_, _runit_, or _s6_, in that it can easily manage non-system processes independently from the system services, but it was designed with unique features that help with more specific use cases.
+
+## Learn the fundamentals
```

Comment:
I'm not a fan of this section here. It seems like a random collection of links, many of which apply to most Linux tools. I think we can assume the target audience knows Linux basics, and things like YAML docs we can link to at the appropriate points (in the tutorial or other docs).